### PR TITLE
export path in shippable and travis

### PR DESCRIPTION
**Pull request by mikedanese**
Build is broken https://github.com/GoogleCloudPlatform/kubernetes/pull/12553#issuecomment-130142456

@uluyol @eparis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ install:
   - go get golang.org/x/tools/cmd/goimports
   - ./hack/travis/install-etcd.sh
   - ./hack/build-go.sh
+  - export PATH=./third_party/etcd:$PATH
```
Comment (uluyol) [inline .travis.yml]: Can we make travis also have $GOPATH/bin? I'm guessing that travis is doing that for us already, but being explicit is better.

Comment (mikedanese) [inline .travis.yml]: done.

**Comment by mikedanese** [comment]
@uluyol can I get an LGTM if you are happy? I'd like to merge on green.

**Comment by uluyol** [comment]
LGTM

**Comment by mikedanese** [comment]
Merging since build is broken

**Pull request merged**